clover.rb

```diff
@@ -477,6 +513,22 @@ def assets
     if api?
       response.json = true
       response.skip_content_security_policy!
+
+      r.rodauth
+      rodauth.check_active_session
+      rodauth.require_authentication
+
+      # Validate request against OpenAPI schema
+      begin
+        schema_validator = SCHEMA_ROUTER.build_schema_validator(r)
+        schema_validator.request_validate(r)
+
+        raise Committee::NotFound, "That request method and path combination isn't defined." if !schema_validator.link_exist?
+      rescue JSON::ParserError => e
+        raise Committee::InvalidRequest.new("Request body wasn't valid JSON.", original_error: e)
+      end
+
+      r.hash_branches("api")
```

Comment:
```suggestion
      # Validate request against OpenAPI schema
      begin
        schema_validator = SCHEMA_ROUTER.build_schema_validator(r)
        schema_validator.request_validate(r)

        raise Committee::NotFound, "That request method and path combination isn't defined." if !schema_validator.link_exist?
      rescue JSON::ParserError => e
        raise Committee::InvalidRequest.new("Request body wasn't valid JSON.", original_error: e)
      end
```

Reply:
Oh huh I guess I did something like that on my own in one of the revisions..